src/scalability/data-sharding.md

```diff
@@ -0,0 +1,23 @@
+# Data sharding
+
+## What is the partition
+
+For fault tolerance, TiKV replicates multiple copies of the same data on different nodes based on Raft. But for that large dataset or high query throughput, all data is managed by one Raft state machine is not efficient and flexible, so we need to break the data up into partitions, also known as `sharding`. And we call a partition here is called a `Region` in TiKV.
+
+With partitioning, we can move partitions to balance the load among all nodes of a cluster. For how to decide which partitions to store on which nodes isn't covered here. It will be discussed later in chapter related to PD.
```

Comment:
```suggestion
With partitioning, we can move partitions to balance the load among all nodes of a cluster. How to decide which partitions to store on which nodes isn't covered here, it will be discussed later in chapter related to PD.
```